### ISSUE TITLE
False error when using placeholder as color value

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/typescript-styled-plugin#60]
From #54 

For the file:

```ts
styled.a`
    color: #${'000'};
`;
```

A false error is reported
